api/api/utils/waveform.py

```diff
@@ -102,7 +102,8 @@ def generate_waveform(file_name, duration):
 
     logger.debug("waveform_generation_started")
 
-    pps = math.ceil(1e6 / duration)  # approx 1000 points in total
+    width = 1e6 if duration > 100 else 1e5
```

Comment:
```suggestion
    # Determine the width of the waveform based on the duration of the audio.
    # The width varies to improve the appearance and "resolution" of the waveform.
    # For long audio files, we set the width to 1,000,000 pixels.
    #
    # For short audio files, we set the width to 100,000 pixels.
    # This prevents the waveform from appearing "stretched out" and sparse.
    width = 1e6 if duration > 100 else 1e5
```

Can we add a comment like this?

Reply:
Added. Thanks!